specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-01-31-preview/applications.json

```diff
@@ -0,0 +1,344 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Chaos Studio Application Resource",
+    "version": "2023-01-31-preview",
+    "description": "Chaos Studio Applications resource type."
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "Operations"
```

Comment:
>Operations [](http://example.com/codeflow?start=15&length=10)

You seem to have removed the 'provider operations' API, is that intentional? I thought its a mandatory API.

Reply:
We dont have additional provider operations api . As this new resource type under existing RP Microsoft.Chaos, we already have operations defined and they are already part of RP.